tutorials/radiant-mlhub-on-demand-training-data.ipynb

```diff
@@ -0,0 +1,1281 @@
+    "DATE_BUFFER = 60\n",
+    "LANDSAT_8_RGB_BANDS = [\"SR_B4\", \"SR_B3\", \"SR_B2\"]  # names of RGB bands from BigEarthNet\n",
+    "BIGEARTHNET_RGB_BANDS = [\"B04\", \"B03\", \"B02\"]  # names of RGB bands from PC Landsat 8\n",
+    "\n",
+    "# Bounding box for demonstration fetching Items over Luxembourg\n",
+    "LUXEMBOURG_AOI = [6.06, 49.58, 6.21, 49.66]  # aoi around Luxembourg"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5d5e31b3-5cde-4a7b-af43-dc194b06d0a0",
+   "metadata": {},
+   "source": [
+    "### Authentication with Radiant MLHub"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5f11e821-b98b-4df1-a26c-826c9bdbec50",
+   "metadata": {},
+   "source": [
+    "Programmatic access to the Radiant MLHub API using the `pystac_client` library requires both the API end-point and an API key. You can obtain an API key for free by registering an account on [mlhub.earth](https://mlhub.earth/). This can be found under `Settings & API Key` from the drop-down once logged in."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f4c9dd60-3abc-464d-af25-4b23c0d2783b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "MLHUB_API_KEY = getpass.getpass(prompt=\"MLHub API Key: \")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ccad2439-109f-4eef-ac3b-dac65f54e3aa",
+   "metadata": {},
+   "source": [
+    "Once you have your API key, you will need to create a default profile by setting up a .mlhub/profiles file in your home directory. You can use the `mlhub configure` command line tool to do this:\n",
+    "\n",
+    "`$ mlhub configure`<br>\n",
+    "API Key: {<i>Enter your API key here</i>}<br>\n",
+    "Wrote profile to /home/jovyan/.mlhub/profiles"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5c77d029-191e-42a5-8250-bc451b80f247",
+   "metadata": {},
+   "source": [
+    "### Configure API connection to Radiant MLHub"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b0480635-eef5-4e3f-847a-5060c409ae4f",
+   "metadata": {},
+   "source": [
+    "This makes a connection to the Radiant MLHub Data Catalog using the API endpoint URL, and the API key from your account."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bf79c301-76df-4158-bf97-3da53552e143",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mlhub_catalog = ps_client.open(\n",
+    "    url=MLHUB_API_URL, parameters={\"key\": MLHUB_API_KEY}, ignore_conformance=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "637734bc-af77-4c81-ab26-98e808de6415",
+   "metadata": {},
+   "source": [
+    "### Fetch label items from BigEarthNet over Luxembourg"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2ba7d7b2-7fed-4412-8c9b-448baad6e595",
+   "metadata": {},
+   "source": [
+    "This helper function below encapsulates the process of querying a STAC API endpoint to fetch an ItemCollection matching query criteria."
```

Comment:
IMO this helper function isn't adding much value over just using `catalog.search` directly. I'd rather teach users how to use `catalog.search`.

Can you remove uses of `search_stac_api` and the `client_catalog.search` instead?

Reply:
Done